elated-egrets/bangazon-internal-site
# J radd employee

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRadd_employee
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. run the tests and make sure everything is passing
1. make sure you are able to add an employee

## What you changed
added the documentation and add employee feature along with tests
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
